crux-metrics/src/crux/metrics/dropwizard/cloudwatch.clj

```diff
@@ -0,0 +1,34 @@
+(ns crux.metrics.dropwizard.cloudwatch
+  (:require crux.metrics.dropwizard)
+  (:import [io.github.azagniotov.metrics.reporter.cloudwatch CloudWatchReporter]
+           [java.io Closeable]
+           [software.amazon.awssdk.services.cloudwatch CloudWatchAsyncClient]
+           [software.amazon.awssdk.regions.providers DefaultAwsRegionProviderChain]
+           [software.amazon.awssdk.regions Region]
+           [software.amazon.awssdk.auth.credentials DefaultCredentialsProvider]
+           [java.util.concurrent TimeUnit]
+           [java.time Duration]
+           [com.codahale.metrics MetricRegistry]))
+
+(defn report ^CloudWatchReporter
+
+  [^MetricRegistry reg {::keys [region dry-run? jvm-metrics? dimensions]}]
+  (let [cwac (-> (CloudWatchAsyncClient/builder)
+                 (.region (if region
```

Comment:
I think these are the defaults - let's make them optional (i.e. `cond->`, as you've done elsewhere)